src-browser/worker_threads.js

```diff
@@ -18,31 +19,40 @@ export class Worker extends globalThis.Worker {
   };
 
   constructor(specifier, options) {
-    super(specifier, { ...(options || {}), type: 'module' });
-    EventEmitter.call(this);
-    this.addEventListener('error', (event) => this.emit('error', event.error || event.message));
-    this.addEventListener('messageerror', (event) => this.emit('messageerror', event.data));
-    this.addEventListener('message', (event) => this.emit('message', event.data));
-    this.postMessage({
+    super();
+    if (options?.eval === true) {
```

Comment:
Avoid optional chaining:

```suggestion
    if (typeof options === 'object' && options !== null && options.eval === true) {
```